### ISSUE TITLE
Cursor position jumps to the end when editing input using state.

### ISSUE THREAD

**Issue by jamiemagique** [zabute/formsy-semantic-ui-react#19]
I think I've found a bug when using this package rather than `semantic-ui-react` directly.

Updating a form input in local state when the input changes, works as expected using `semantic-ui-react` but with `formsy-semantic-ui-react` when updating an input value when the cursor is not at the end it enters the first character and then causes a rerender, jumping the cursor to the end of the input.

eg.
1. Input value is 'abc'
2. Attempting to insert '123' between the characters 'ab' results in 'a1bc23'

See below example source code:
```
import React, { Component } from 'react';
import { Form } from 'semantic-ui-react';
// import { Form } from 'formsy-semantic-ui-react';

class TestForm extends Component {
  state = {
    name: '',
  };

  handleChange = (e, { name, value }) => this.setState({ [name]: value });

  render() {
    return (
      <Form>
        <Form.Input name="name" label="Name" onChange={this.handleChange} />
      </Form>
    );
  }
}

export default TestForm;
```

Am I doing something wrong or is this a valid bug? 

Many thanks in advance for any help.
Jamie

**Comment by zabute** [comment]
@jamiemagique Good catch. I just published ```0.2.11``;  should be fixed now. Let me know if you any more problems. Thanks

**Comment by jamiemagique** [comment]
@zabute Nice one, `0.2.11` has fixed the issue!
